fusion/python/sample.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
+"""
+Yelp Fusion API code sample.
+
+This program demonstrates the capability of the Yelp Fusion API
+by using the Search API to query for businesses by a search term and location,
+and the Business API to query additional information about the top result
+from the search query.
+
+Please refer to http://www.yelp.com/developers/v3/documentation for the API
+documentation.
+
+This program requires the Python requests library, which you can install via:
+`pip install -r requirements.txt`.
+
+Sample usage of the program:
+`python sample.py --term="bars" --location="San Francisco, CA"`
+"""
+from __future__ import print_function
+
+import argparse
+import json
+import pprint
+import requests
+import sys
+import urllib
+
+
+# This client code can run on Python 2.x or 3.x.  Your imports can be
+# simpler if you only need one of those.
+try:
+    # For Python 3.0 and later
+    from urllib.error import HTTPError
+    from urllib.parse import quote
+    from urllib.parse import urlencode
+except ImportError:
+    # Fall back to Python 2's urllib2 and urllib
+    from urllib2 import HTTPError
+    from urllib import quote
+    from urllib import urlencode
+
+
+# OAuth credential placeholders that must be filled in by users.
+CLIENT_ID = None
+CLIENT_SECRET = None
+
+
+# API constants, you shouldn't have to change these.
+API_HOST = 'api.yelp.com'
```

Comment:
Guessing it works from the interactive testing but should we still prepend https:// so there are no questions?